packages/playwright-test/src/tsxTransform.ts

```diff
@@ -23,6 +23,42 @@ export default declare(api => {
   return {
     name: 'playwright-debug-transform',
     visitor: {
+      Program(path) {
+        path.setData('pw-imports', []);
+      },
+
+      ImportDeclaration(path) {
+        // Non-JSX transform, replace
+        //   import Button from './ButtonVue.vue'
+        //   import { Card as MyCard } from './Card.vue'
+        // with
+        //   const { Button } = { Button: 'Button' }, { MyCard } = { MyCard: 'Card' };
+        const importNode = path.node;
+        if (!t.isStringLiteral(importNode.source))
+          return;
+        if (!importNode.source.value.endsWith('.vue') && !importNode.source.value.endsWith('.svelte'))
+          return;
+        const components = path.parentPath.getData('pw-components');
+        for (const specifier of importNode.specifiers) {
+          if (t.isImportDefaultSpecifier(specifier))
+            components.set(specifier.local.name, specifier.local.name);
+          if (t.isImportSpecifier(specifier) && t.isIdentifier(specifier.imported))
+            components.set(specifier.local.name, specifier.imported.name);
+        }
+
+        const importNodes = (path.parentPath.node as t.Program).body.filter(i => t.isImportDeclaration(i));
+        const isLast = importNodes.indexOf(importNode) === importNodes.length - 1;
```

Comment:
I think this will not work when last import is not a vue/svelte import or is not a string literal.